scripts/install-celo.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+set -euo pipefail
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+
+KEEP_CORE_PATH=$PWD
+KEEP_CORE_SOL_PATH="$KEEP_CORE_PATH/solidity"
+
+# Defaults, can be overwritten by env variables/input parameters
+CONFIG_DIR_PATH_DEFAULT="$KEEP_CORE_PATH/configs"
+NETWORK_DEFAULT="local"
+KEEP_CELO_PASSWORD=${KEEP_CELO_ACCOUNT_PASSWORD:-"password"}
```

Comment:
`KEEP_CELO_PASSWORD=${KEEP_CELO_PASSWORD:-"password"}`?
This would be aligned with `KEEP_ETHEREUM_PASSWORD` from `install.sh`.

Reply:
done